includes/class-wc-customer-download.php

```diff
@@ -290,15 +290,21 @@ public function set_download_count( $value ) {
 	 * @throws Exception When permission ID is invalid.
 	 * @param int    $user_id         Id of the user performing the download.
 	 * @param string $user_ip_address IP Address of the user performing the download.
+	 * @param bool   $range_download  Whether the download is a partial/range download.
 	 */
-	public function track_download( $user_id = null, $user_ip_address = null ) {
+	public function track_download( $user_id = null, $user_ip_address = null, $range_download = false ) {
```

Comment:
If track_download does not run for `range_download` (which I agree is the correct handling) why is this passed through?

Could this check not go in the download handler?

Reply:
Good point, I initially thought that we might be updating the ranges in the `wp_wc_download_log`, but since that's not the way it ended up, it can be removed altogether.